packages/orbit-components/src/ChoiceTile/styled.ts

```diff
@@ -0,0 +1,105 @@
+import styled, { css } from "styled-components";
+
+import { StyledText } from "../Text";
+import defaultTheme from "../defaultTheme";
+import mq from "../utils/mediaQuery";
+import { left } from "../utils/rtl";
+import { StyledLabel as RadioLabel } from "../Radio";
+import { StyledLabel as CheckboxLabel } from "../Checkbox";
+
+export const StyledChoiceTile = styled.div<{ selected?: boolean }>`
+  ${({ theme, selected }) => css`
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    border-radius: ${theme.orbit.borderRadiusNormal};
+    padding: ${theme.orbit.spaceSmall};
+    box-shadow: ${theme.orbit.boxShadowAction};
+    cursor: pointer;
+    outline: ${selected && `2px solid ${theme.orbit.paletteBlueNormal}`};
+    transition: box-shadow ${theme.orbit.durationFast} ease-in-out;
+
+    ${mq.tablet(css`
+      padding: ${theme.orbit.spaceMedium};
+    `)}
+
+    &:hover {
+      box-shadow: ${theme.orbit.boxShadowActionActive};
+    }
+  `}
+`;
+
+StyledChoiceTile.defaultProps = {
+  theme: defaultTheme,
+};
+
+export const StyledChoiceTileHeading = styled(StyledText)`
+  ${({ theme }) => css`
+    font-size: ${theme.orbit.fontSizeHeadingTitle3};
```

Comment:
In Figma, it has two variations, with font-size 18px and line-height 24px and font size 16px and line-height 20px. You have a font-size 16px and a line height 24px. Not sure why it's like that, but there is something wrong, I think

Reply:
On the sync, Will talked about the possibility of only having one size variation, and not two. Until the decision is made, I have implemented only one, as agreed

Reply:
Okay, but if the font-size 16px, should not it have 20px line-height?

Reply:
Hmm, yes, but then our tokens are a bit misaligned. The `fontSizeHeadingTitle3` is 16px and `lineHeightHeadingTitle3` is 24px. But in this design the conjugation is different. I'll sync with designers.

Reply:
Agreed with Will to keep it as is (with Title3 tokens) as the font-size for Title3 is expected to grow to 18px. The line-height is already correct.